web-components/src/components/button/tokens/lm-button-tokens.js

```diff
@@ -49,13 +49,13 @@ const button = {
     hover: {
       "bg-color": {
         light: colors.gray[30].name,
-        dark: colors.gray[70].name
+        dark: colors.gray["95"].name
```

Comment:
Any reason for the quotation marks and not just `colors.gray[95].name`?

Reply:
you should only need quotes when entering "05", otherwise, you can pas in just the number value.

Reply:
Changed